.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,34 @@
-Please read and understand everything below
-**Do not delete any text other than where you are instructed.**
+<!-- Please read and understand everything below
+**Do not delete any text other than where you are instructed.** -->
```

Comment:
other than where you are instructed.  => other **than the ones you are intsructed to**.

Reply:
@saarthakchats Please be careful while editing. That was not the suggested change. It is grammatically incorrect.